org/cloudfoundry.yml

```diff
@@ -3386,11 +3389,21 @@ orgs:
         - danielfor
         - klakin-pivotal
         - camilalonart
-        - nouseforaname
-        - malba
         members:
+        - aramprice
```

Comment:
To be propagated: https://github.com/cloudfoundry/community/commit/24624505333be82b98c8aa7168ab8a0c47dd23dd